docs/index.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: GreenX Library
+tagline: GreenX
+description: Library for Many-body Greens Functions on HPC
+---
+
+# GreenX
+
+With [GitHub Pages](https://pages.github.com), you just write things in
+[Markdown](https://daringfireball.net/projects/markdown/),
+[GitHub](https://github.com) hosts the site for you, and you just push
+material to your GitHub repository with `git add`, `git commit`, and
+`git push`.
+
+The sites use [Jekyll](https://jekyllrb.com/), a
+[ruby](https://www.ruby-lang.org/en/) [gem](https://rubygems.org/), to
+convert Markdown files to html, and this part is done
+automatically when you push the materials to the `gh-pages` branch
+of a GitHub repository.
```

Comment:
Note to self, this should be removed/replaced. It was just a placeholder to impact so info on Github pages